docs/index.md

```diff
@@ -14,6 +14,33 @@ To install the current release (`{{current_release}}`) of Sinon:
 npm install sinon
 ```
 
+### Setting up access
+
+#### Node and CommonJS build systems
+
+```javascript
+const sinon = require('sinon');
```

Comment:
Use `var` here. We are targetting ES5 engines, which don't support that syntax.

Reply:
Gotcha... Fixed...